bittide/src/Bittide/ClockControl/Si539xSpi.hs

```diff
@@ -301,12 +306,13 @@ si539xSpiDriver SNat incomingOpS miso = (fromSlave, decoderBusy, spiOut)
       | otherwise = satPred SatZero idleCycles
 
     nextState
-      | commandAcknowledged && not spiBusy =
-        DriverState nextPage nextAddress nextOp False maxBound
+      | commandAcknowledged && not spiBusy && isNothing storedBytes=
+        DriverState nextPage nextAddress nextOp False (fmap resize outBytes) maxBound
```

Comment:
```suggestion
      | commandAcknowledged && not spiBusy && isNothing storedBytes =
        DriverState nextPage nextAddress nextOp False (fmap resize outBytes) maxBound
```

Reply:
Turns out this is actually multiple bytes (`BitVector 16`). The resize just takes the byte at the LSBs.